web3/_utils/filters.py

```diff
@@ -90,10 +90,7 @@ def construct_event_filter_params(
             )
         topic_set = topics
 
-    if len(topic_set) == 1 and is_list_like(topic_set[0]):
```

Comment:
I assume this was a flake8 check that flagged? do we have it tested somewhere?

Reply:
No, this is what Stu's original comment was. The type of `topic_set` in the function definition is `Sequence[HexStr]`, but here we had this "but maybe it's a nested list sometimes?" check. The return type of `construct_event_topic_set` is `List[HexStr]`, so correct behavior seems to be to assume it will be a `Sequence[HexStr]`, then if  some edge case crops up that returns a nested list, fix that as a bug and add a test there.

Reply:
Okay, sounds good!